csharp/AppEncryption/AppEncryption/Persistence/DynamoDbMetastoreImpl.cs

```diff
@@ -71,19 +82,22 @@ private interface IEndPointStep
             /// <summary>
             /// Adds EndPoint config to the AWS DynamoDb client.
             /// </summary>
+            ///
             /// <param name="endPoint">the service endpoint either with or without the protocol.</param>
             /// <param name="signingRegion">the region to use for SigV4 signing of requests (e.g. us-west-1).</param>
-            /// <returns>The current <code>IBuildStep</code> instance.</returns>
+            /// <returns>The current <see cref="IBuildStep"/> instance.</returns>
             IBuildStep WithEndPointConfiguration(string endPoint, string signingRegion);
         }
 
         private interface IRegionStep
```

Comment:
All these interfaces should be public to keep builder interfaces consistent across the SDK

Reply:
discussed as not a part of this PR